services/common/rs_server_common/data_retrieval/eodag_provider.py

```diff
@@ -0,0 +1,80 @@
+"""EODAG Provider."""
+from pathlib import Path
+
+from eodag import EODataAccessGateway, EOProduct
+
+from .provider import CreateProviderFailed, Product, Provider, TimeRange
+
+
+class EodagProvider(Provider):
+    """An EODAG provider.
+
+    It uses EODAG to provide data from external sources.
+    """
+
+    def __init__(self, config_file: str, provider: str):
```

Comment:
Why do you remove the EodagConfiguration ?

Reply:
I don't see the real usage of a class that store 2 strings. It can be a Tuple or a hashable.